Purchases/Attribution/RCAttributionFetcher.m

```diff
@@ -13,23 +13,15 @@
 #import "RCIdentityManager.h"
 #import "RCBackend.h"
 #import "RCAttributionData.h"
+#import "RCSystemInfo.h"
 @import PurchasesCoreSwift;
 
-
-@protocol FakeAdClient <NSObject>
-
-+ (instancetype)sharedClient;
-- (void)requestAttributionDetailsWithBlock:(RCAttributionDetailsBlock)completionHandler;
-
-@end
-
-
-@protocol FakeASIdentifierManager <NSObject>
-
-+ (instancetype)sharedManager;
-
-@end
```

Comment:
moved all of this stuff to the new `AttributionTypeFactory`, because otherwise it was impossible to test, since we were directly depending on the class names and static attributes.